xinference/model/llm/llm_family.json

```diff
@@ -494,6 +494,15 @@
     ],
     "model_description": "ChatGLM3 is the third generation of ChatGLM, still open-source and trained on Chinese and English data.",
     "model_specs": [
+      {
+        "model_format": "ggmlv3",
+        "model_size_in_billions": 6,
+        "quantizations": [
+          "q4_0"
+        ],
+        "model_id": "Xorbits/chatglm2-6B-GGML",
```

Comment:
should be `chatglm3`?

The filename template also needs to be fixed.

Reply:
Fixed.